### DISCUSSION TITLE
timeseries_SET_ERA5_ramp_demErr.h5

### DISCUSSION THREAD

**Discussion by irmayolii** [insarlab/MintPy#1217] [Q&A]
How can I get timeseries_SET_ERA5_ramp_demErr.h5 file?, because after I run all the script I just got timeseries.h5, and what's the different between timeseries_ERA5 and timeseries.h5?

**Comment by welcome** [comment]
👋  Thanks for opening your first issue here! Please filled out the template with as much details as possible. We appreciate that you took the time to contribute!
Make sure you read our [contributing guidelines](https://github.com/insarlab/MintPy/blob/master/docs/CONTRIBUTING.md).
